server/exec_functests

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Very simple server functional test script
+#
+# Install the Pbench package inside a temporary virtual environment, activate
+# that virtual environment to get access to the pbench.client package (we have
+# no other dependencies here), run the server functional tests, and then
+# deactivate the virtual environment.
+# 
+virtualenv -p /usr/bin/python3 /var/tmp/venv
+source /var/tmp/venv/bin/activate
```

Comment:
I'd be inclined to configure the location separately, and perhaps to allow it to be specified externally.  E.g.,
```
VIRT_ENV=${VIRT_ENV:-/var/tmp/venv}
virtualenv -p /usr/bin/python3 ${VIRT_ENV}
source ${VIRT_ENV}/bin/activate
```

Reply:
Seems like overkill, and more parameters to juggle. Definitely not interested in dealing with that here.